introduction.md

```diff
@@ -0,0 +1 @@
+Hi there, Dojo. I am Jay, and I'm at your service. Give me all I can handle!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.